Include/pythread.h

```diff
@@ -20,6 +20,25 @@ PyAPI_FUNC(unsigned long) PyThread_start_new_thread(void (*)(void *), void *);
 PyAPI_FUNC(void) _Py_NO_RETURN PyThread_exit_thread(void);
 PyAPI_FUNC(unsigned long) PyThread_get_thread_ident(void);
 
+#if !defined(Py_LIMITED_API)
```

Comment:
If it's excluded from the limited API, you should add it to Include/cpython/pythread.h

Reply:
I'll do. `PyThread_start_new_thread` is not even documented officially, so I'm surprised it's part of the limited API...

Reply:
Well, in the past, we didn't pay much attention about limited VS non-limited API. Today, the fact that it's not documented is not enough to justify to move it the non-limited API: it should stay in limited API. You can document it (and add tests if needed) obviously :-)

Now we are paying more attention and we are trying to avoid mistakes done in the past.